### PR TITLE
Expand heat map

### PR DESCRIPTION
**Pull request by panerorenn9541**
- Reduce the size of heat to show data more specifically
- Add option to toggle heatmap on/off
- Allow user to display active, confirmed, deaths, recovered, and per-capita case heatmaps
- Drop down menu for each heatmap type
- Confirmed cases heatmap initially displayed

### CODE REVIEW

**Comment by panerorenn9541** [comment]
Screenshot 2020-08-04 at 3 31 13 PM

Heat size is smaller

**Comment by panerorenn9541** [comment]
Screenshot 2020-08-04 at 3 31 27 PM

User can choose different data to display as heatmap (such as per-capita cases displayed here)

**Comment by panerorenn9541** [comment]
Screenshot 2020-08-04 at 3 31 44 PM

User can toggle heat map off with button in top left corner, next to the heatmap chooser

**Review by alsile — APPROVED** [review]

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -42,5 +42,14 @@
       <p id="displayDeaths"></p>
       <p id="displayRecovered"></p>
     </div>
+
+    <button type="button" id="toggle-heat">Toogle Heatmap</button>
```
Comment (kinsaurralde) [inline src/main/webapp/index.html]: Toggle

**Review by kinsaurralde — APPROVED** [review]

**Review by huzcn — APPROVED** [review]
Great! As a user, I would feel it's more helpful to choose the granularity of the heatmap, e.g. a zoom in/out bar to change contrast of the color. But this can be in a follow-on step if you want to add.

**Review by v01dstar — APPROVED** [review]
LGTM

**Pull request merged**